src/Stratis.Bitcoin.IntegrationTests/RPC/RPCTests.cs

```diff
@@ -214,6 +214,27 @@ public void CanGenerateByStringArgs()
             Assert.StartsWith("[" + Environment.NewLine + "  \"", resp);
         }
 
+        /// <summary>
+        /// Tests that RPC method 'SendRawTransaction' can be called with a new transaction.
+        /// </summary>
+        [Fact]
+        public void SendRawTransaction()
+        {
+            var tx = new Transaction();
+            tx.Outputs.Add(new TxOut(Money.Coins(1.0m), new Key()));
+            this.rpcTestFixture.RpcClient.SendRawTransaction(tx);
```

Comment:
What is this test checking? there are not asserts.

Reply:
That the RPC method executes without throwing an exception. It basically broadcasts the transaction to the mempool I think, so maybe I could some how check that it was received in mempool. Not sure exactly what else to do here. 

The transaction should get rejected cause it isn't signed

Reply:
ok yeah you could ask the trx form the node and get back null but it wont help much so I guess like this is ok.